const.gen.go

```diff
@@ -1963,3 +1963,193 @@ const (
 	StreamEventError    StreamEventType = 4
 	StreamEventHangup   StreamEventType = 8
 )
+
+// ErrorLevel as declared in libvirt/virterror.h:44
```

Comment:
I don't want to make these error types part of go-libvirt's public API. Can you adjust the yaml file to make them private? (I think the unexport transform in c-for-go will work here.) This applies to all the types from virterror.h.